### DISCUSSION TITLE
Fade-out option for Taiko Dons/Kats

### DISCUSSION THREAD

**Discussion by Shavixinio** [ppy/osu#29497] [Ideas]
Basically instead of the Dons/Kats going to the left like on the screenshot below:
![image](https://github.com/user-attachments/assets/10e32e49-7684-436f-9a8d-04b42cc848d2)
They fade out similarly like in stable with HUD turned off instead of going to the left

**Comment by peppy** [comment]
This has been brought up multiple times before. Definitely something we'll want to add.
